### PR TITLE
Fix autosde cloud volume editing

### PR DESCRIPTION
**Pull request by llivne**
If we try to modify cloud volume managed by autosde storage manager we get an exception. 
This PR enables size and name modifications for volumes managed under autosde.

links
-----
https://github.com/ManageIQ/manageiq-ui-classic/pull/7511 - bug fix in the UI preventing autosde cloud volume editing

### CODE REVIEW

File: app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb [hunk 1]
```diff
@@ -31,4 +31,22 @@ def raw_delete_volume
...
+  # ================= edit  ================
+
+  # override the function in manageiq\app\models\cloud_volume\operations.rb
+  def validate_update_volume
+    {:available => true, :message => nil}
+  end
+
+  def raw_update_volume(options = {})
+    ems = ext_management_system
+    vol_to_update = ems.autosde_client.VolumeUpdate(
```
Comment (gregoryb-at-ibm) [inline app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb:44]: This isn't a volume to update, what updates we want to make. An appropriate name could be `volume_update`, `update_details`, `things_to_change`, etc..

**Comment by gregoryb-at-ibm** [comment]
I think we can disregard the rubocop complaints about the generated code for now.

File: app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb (hunk 1 above)
Comment (llivne) [reply app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb]: agree. changed to update_details

**Comment by llivne** [comment]
> I think we can disregard the rubocop complaints about the generated code for now.

I will open a new ticket about it

**Comment by agrare** [comment]
> I think we can disregard the rubocop complaints about the generated code for now.

Yes, you can add a rubocop exclude for the openapi client:
```
.rubocop_local.yml

AllCops:
  Exclude:
  - lib/autosde_oas_client/generated/**/*
```

Should do the trick

**Comment by llivne** [comment]
> > I think we can disregard the rubocop complaints about the generated code for now.
> 
> Yes, you can add a rubocop exclude for the openapi client:
> 
> ```
> .rubocop_local.yml
> 
> AllCops:
>   Exclude:
>   - lib/autosde_oas_client/generated/**/*
> ```
> 
> Should do the trick

great suggestion. I added it.

**Comment by agrare** [comment]
Marking `kasparov/yes` because https://github.com/ManageIQ/manageiq-ui-classic/pull/7511 was already backported /cc @simaishi

**Comment by simaishi** [comment]
Kasparov backport details:

```
$ git log -1
commit 4ae97c88dd64e57ca8c949c2d36f0a01e237015f
Author: gregoryb-at-ibm <68283004+gregoryb-at-ibm@users.noreply.github.com>
Date:   Mon Nov 16 15:44:00 2020 +0200

    Merge pull request #47 from Autosde/fix_edit_cloud_volume

    Fix autosde cloud volume editing

    (cherry picked from commit f98eff78a80c98291b0fe7d270f4fda73c3310c1)
```

**Pull request merged**